src/components/Feed/FeedPage.vue

```diff
@@ -83,4 +90,24 @@ const formatSource = (source: any) => {
     }
 }
 
+// Sort feed functions
+const sortedFeed = computed(() => {
+    console.log('Sorting by:', sortType.value);
+    switch (sortType.value) {
+        case 'Newest':
+            return [...feed.value].sort((a, b) => new Date(b.time).getTime() - new Date(a.time).getTime());
```

Comment:
feed.value is already an array...

Might be able to optimize in future by just saying feed.value.sort(...)